catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorTest.java

```diff
@@ -186,7 +186,7 @@ public void testDirectoryMonitorWithParameters() throws Exception {
                 .get(0);
         assertThat(routeDefinition.toString(), containsString(
                 "SetHeader[" + Constants.ATTRIBUTE_OVERRIDES_KEY
-                        + ", simple{Simple: test1=someParameter1,test2=someParameter2}"));
+                        + ", {{test1=someParameter1, test2=someParameter2}}"));
```

Comment:
I dug as deep as I think I can and it just ends up with me relying on a `toString` in `SetHeaderDefinition` instead...